docs/deployment/local-deployment.rst

```diff
@@ -142,11 +142,12 @@ If we don't have it already, we need to create a new file ``application.yml``. W
     # ... other configuration
 
     repository:
-        type: 5
-        blazegraph:
-            url: http://blazegraph:8080/blazegraph
+        type: 4
+        graphDb:
+            url: http://graphdb:7200
+            repository: fdp
 
-We now need to update our ``docker-compose.yml`` file, we add a new volume for the ``fdp`` and add ``blazegraph`` service. We can also expose port ``8080`` for Blazegraph so we can access its user interface.
+We now need to update our ``docker-compose.yml`` file, we add a new volume for the ``fdp`` and add ``graphdb`` service. We can also expose port ``7200`` for GraphDB so we can access its user interface.
```

Comment:
Perhaps this is also a good time to update all Docker references to reflect modern practices:

- preferred [default][1] filename is `compose.yml` (`docker-compose.yml` is support for backward compat.)
- the `version` element is [obsolete][2]
- the cli [command][3] is now `docker compose`, instead of `docker-compose`

[1]: https://docs.docker.com/compose/intro/compose-application-model/#the-compose-file
[2]: https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete
[3]: https://docs.docker.com/reference/cli/docker/compose/

Reply:
I'll try to incorp those changes while I'm at it.